tools/python3/scorep.py

```
@@ -0,0 +1,117 @@
+if __name__ == "__main__":
+    if len(sys.argv) == 1:
+        print("usage: $function_name -h")
+        print(
+            "available functions:\n\t"
+            + "\n\t".join([k for k, v in globals().items() if k.startswith("print_")]),
+        )
+    elif len(sys.argv) > 1:
+        fn = sys.argv[1]
+        sys.argv = [sys.argv[0]] + sys.argv[2:]
+        if fn not in globals():
+            raise ValueError("requested function does not exist")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)